### PR TITLE
[zh]Fix tools/kubeadm/high-availablility.md display format

### PR DESCRIPTION
**Pull request by FOWind**
Related issues: #31499 

Placing the comment section outside the `{{< note >}}{{< /note >}}` tag. Making the page display with the right format

Before:
image

After:
image

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
I suspect that the actual issue here is with the number of spaces at the start of lines. Changing the whitespace might be a better fix.

**Comment by tengqm** [comment]
This is annoying and it is one of the reasons I don't like these shortcodes.
Maybe indent the bullets by three spaces can help, but ... no one knows.

**Comment by sftim** [comment]
4 space indent is CommonMark Markdown for a code block. We should avoid lines that are indented by 4 spaces (even for code blocks - I prefer explicit blocks).

**Review by Arhell — COMMENTED** [review]
/check-cla

**Comment by FOWind** [comment]
> This is annoying and it is one of the reasons I don't like these shortcodes. Maybe indent the bullets by three spaces can help, but ... no one knows.

I have tested by use 3 spaces indent, It will keep the wrong format. 

3 spaces indent code:
image

display with wrong format:
image

**Comment by FOWind** [comment]
> I suspect that the actual issue here is with the number of spaces at the start of lines. Changing the whitespace might be a better fix.

I have tested by use 3 spaces indent, it still display different between placing the comment outside.

Code:
image

Result:
image

further more, change 4 spaces indent to 3 spaces indent may introduce large code change.

**Comment by tengqm** [comment]
/approve cancel
looks like there are some contents not synced.
We will not merge partial fixes as per stated in the [guideline](https://kubernetes.io/zh/docs/contribute/localization_zh/#%E6%9B%B4%E6%96%B0%E8%AF%91%E6%96%87). To find out what are the upstream changes to be synced, do this **on the main branch**:

```shell
./scripts/lsync.sh content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md
```

**Comment by chenxuc** [comment]
Just notice that the doc is not fully synced.

/lgtm cancel

**Comment by FOWind** [comment]
Working on sync new docs

**Comment by FOWind** [comment]
@tengqm @chenxuc 
Have finished on syncing from main doc.

File: content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md [hunk 1]
```diff
@@ -185,182 +275,181 @@ option. Your cluster requirements may need a different configuration.
...
 
 <!--
-1.  Type the following and watch the pods of the control plane components get started:
+1. Apply the CNI plugin of your choice:
+   [Follow these instructions](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network) to install the CNI provider. Make sure the configuration corresponds to the Pod CIDR specified in the kubeadm configuration file (if applicable).
 -->
-3.  输入以下内容，并查看控制平面组件的 Pods 启动：
+2. 应用你所选择的 CNI 插件：
+   [请遵循以下指示](/zh/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#pod-network)
+   安装 CNI 提供程序。如果适用，请确保配置与 kubeadm 配置文件中指定的 Pod
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md:390]: 在这个语境下面，provider 可以翻译为驱动。

File: content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md [hunk 2]
```diff
@@ -185,182 +275,181 @@ option. Your cluster requirements may need a different configuration.
...
-    sudo kubeadm join 192.168.0.200:6443 --token 9vr73a.a8uxyaju799qwdjv --discovery-token-ca-cert-hash sha256:7c2e69131a36ae2a042a339b33381c6d0d43887e2de83720eff5359e26aec866 --control-plane --certificate-key f8902e114ef118304e561c3ecd4d0b543adc226b7a07f675f56564185ffe0c07
-    ```
+   ```sh
+   sudo kubeadm join 192.168.0.200:6443 --token 9vr73a.a8uxyaju799qwdjv --discovery-token-ca-cert-hash sha256:7c2e69131a36ae2a042a339b33381c6d0d43887e2de83720eff5359e26aec866 --control-plane --certificate-key f8902e114ef118304e561c3ecd4d0b543adc226b7a07f675f56564185ffe0c07
+   ```
 
-    - 这个 `--control-plane` 命令通知 `kubeadm join` 创建一个新的控制平面。
-    - `--certificate-key ...` 将导致从集群中的 `kubeadm-certs` Secret 下载
-      控制平面证书并使用给定的密钥进行解密。
+   - 这个 `--control-plane` 命令通知 `kubeadm join` 创建一个新的控制平面。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md:450]:
```suggestion
   - 这个 `--control-plane` 标志通知 `kubeadm join` 创建一个新的控制平面节点。
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md [hunk 3]
```diff
@@ -185,182 +275,181 @@ option. Your cluster requirements may need a different configuration.
...
+   - `--certificate-key ...` 将导致从集群中的 `kubeadm-certs` Secret 下载
+     控制平面证书并使用给定的密钥进行解密。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md:452]:
```suggestion
   - `--certificate-key ...` 将导致从集群中的 `kubeadm-certs` Secret
     下载控制平面证书并使用给定的密钥进行解密。
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md [hunk 4]
```diff
@@ -377,124 +466,129 @@ in the kubeadm config file.
...
+   ```
 
-    - Replace the value of `CONTROL_PLANE` with the `user@host` of the first control-plane machine.
+   - Replace the value of `CONTROL_PLANE` with the `user@host` of the first control-plane machine.
 -->
 ### 设置 ectd 集群
 
-1.  按照 [这些指示](/zh/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/) 
-    去设置 etcd 集群。
+1. 按照 [这些指示](/zh/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/)
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md:486]:
```suggestion
1. 按照[这些指示](/zh/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/) 
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md [hunk 5]
```diff
@@ -377,124 +466,129 @@ in the kubeadm config file.
...
 -->
 ### 设置 ectd 集群
 
-1.  按照 [这些指示](/zh/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/) 
-    去设置 etcd 集群。
+1. 按照 [这些指示](/zh/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/) 
+   去设置 etcd 集群。
 
-1.  根据[这里](#manual-certs)的描述配置 SSH。
+1. 根据 [这里](#manual-certs) 的描述配置 SSH。
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md:489]:
```suggestion
1. 根据[这里](#manual-certs)的描述配置 SSH。
```

File: content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md [hunk 6]
```diff
@@ -377,124 +466,129 @@ in the kubeadm config file.
...
+       certFile: /etc/kubernetes/pki/apiserver-etcd-client.crt
+       keyFile: /etc/kubernetes/pki/apiserver-etcd-client.key
+   ```
+   <!--
+   The difference between stacked etcd and external etcd here is that the external etcd setup requires
+   a configuration file with the etcd endpoints under the `external` object for `etcd`.
+   In the case of the stacked etcd topology this is managed automatically.
+   -->
+   {{< note >}}
+   这里的内部（stacked） etcd 和外部 etcd 之前的区别在于设置外部 etcd
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/tools/kubeadm/high-availability.md:551]:
```suggestion
   这里的堆叠（stacked）etcd 和外部 etcd 之前的区别在于设置外部 etcd
```

**Pull request merged**